src/components/NFTViewOptionList.vue

```diff
@@ -82,14 +85,14 @@
       </template>
       <template v-else>
         <ButtonV2
-          v-for="contentUrl in normalizedContentURLs"
+          v-for="(contentUrl, index) in normalizedContentURLs"
           :key="contentUrl"
           class="mt-[12px] w-full"
           preset="outline"
-          :text="getContentUrlButtonText(contentUrl)"
```

Comment:
Should not change this?

Reply:
oops ~ misprint!
[🎨 Add 'index' to query for retrieving specific 'fileSrc'](https://github.com/likecoin/liker-land/pull/1525/commits/9179f8dc3dc27a69ddd42c3e5eb706f4bb2262e5)